sources/Chizkuny/Chizkuni_Devarim_parse.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+# for a script located two directories below this file
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+sys.path.append("/Users/Eytan/Desktop/sefaria/Sefaria-Data/sources")
+
+from local_settings import *
+from fuzzywuzzy import fuzz
+
+sys.path.insert(0, SEFARIA_PROJECT_PATH)
+os.environ['DJANGO_SETTINGS_MODULE'] = "local_settings"
+#from sefaria.model import *
+from functions import *
+import re
+
+def get_parsed_text():
+    with open("Chizkuni_Devarim.txt",'r') as myfile:
+        lines = myfile.readlines()
+    current_perek = 0
+    current_pasuk = 0
+    devarim_array = make_perek_array("Devarim")
+    for line in lines:
+        if "@11" in line:
+            current_perek = getGematria(line.replace("@11",""))
+        elif "@22" in line:
+            current_pasuk = getGematria(line.replace("@22",""))
+        else:
+            if not_blank(line):
+                print line
+                devarim_array[current_perek-1][current_pasuk-1].append(line)
+                #print "ADDED! " + str(current_perek-1)+" "+str(current_pasuk-1)+" "+str(devarim_array[current_perek-1][current_pasuk-1])
+                #print "LAST!" + str(devarim_array[0][0])
+
+    for pindex, perek in enumerate(devarim_array):
+        for qindex, pasuk in enumerate(perek):
+            for comment in pasuk:
+                print str(pindex)+" "+str(qindex)+" "+ comment
+    return devarim_array
+
+def make_perek_array(book):
```

Comment:
Not necessary. Use a JaggedArray instance (located in `sefaria.datatype.jagged_array` withing Sefaria-Project)